test/pkg/test/pod_manager.go

```diff
@@ -56,9 +56,9 @@ func (env *Framework) PodExec(namespace string, podName string, cmd string, prin
 	stdoutStr := stdout.String()
 	stderrStr := stderr.String()
 	if printOutput {
-		log.Println("stdout: ", stdoutStr)
-		log.Println("stderr: ", stderrStr)
-		log.Println("err: ", err)
+		env.log.Infoln("stdout: ", stdoutStr)
+		env.log.Infoln("stderr: ", stderrStr)
+		env.log.Infoln("err: ", err)
```

Comment:
nit: single log statement? you can do either log.Infof("stdout: %s, stderr: %s...") or structured logging with log.Infow("pod exec output", "stdout", stdout, "stderr", stderr)

Reply:
Changed, now it print:
`{"level":"info","ts":1693523160.295648,"logger":"framework","caller":"test/pod_manager.go:22","msg":"PodExec() [namespace: default] [podName: grpcurl-runner] [command: /grpcurl -proto /protos/addsvc.proto -proto /protos/grpcbin.proto -proto /protos/helloworld.proto   -d '{ \"a\": \"Str1\", \"b\": \"Str2\"}' bardw-5-e9jmw297uy-non-default-078b79829cc6f5a70.7d67968.vpc-lattice-svcs.us-west-2.on.aws:443 addsvc.Add/Concat] \n"}`
if `printOutput==true`
`{"level":"info","ts":1693523160.548332,"logger":"framework","caller":"test/pod_manager.go:59","msg":"pod exec output","stdout":"{\n  \"v\": \"Str1Str2\"\n}\n","stderr":""}`